server/models/Item.js

```diff
@@ -5,7 +5,16 @@ const schema = new mongoose.Schema({
   description: { type: String, required: true },
   imgUrl: { type: String },
   rating: { type: Number },
-  userId: {type: String}
-})
+  userId: { type: String },
+});
 
-module.exports = Item = mongoose.model("items", schema);
+schema.set("toJSON", {
+  virtuals: true,
+  transform: (document, returnedObject) => {
+    returnedObject.id = returnedObject._id.toString();
+    delete returnedObject._id;
+    delete returnedObject.__v;
+  },
+});
+
```

Comment:
Is this a preferred convention in processing MongoDB items? I thought the database entries are valid JSON by default. Are they not?

Reply:
Frontend doesn't really need _v info, its just for version revisions of the document for the database.  Id is also more generic naming than _id in frontend. But this is not really a must change if not needed.